pkgs/development/tools/rgp/default.nix

```diff
@@ -59,6 +60,12 @@ stdenv.mkDerivation rec {
     mkdir -p $out/opt/rgp $out/bin
     cp -r . $out/opt/rgp/
 
+    for script in $out/opt/rgp/scripts/*; do
+      substituteInPlace $script \
+        --replace "/bin/bash" "${bash}/bin/bash"
```

Comment:
Is this a shebang? If yes we should use patchShebang.

Reply:
Thanks, it’s now using `patchShebangs`.